pymc/sampling/jax.py

```diff
@@ -74,6 +74,15 @@ def assert_fn(value, *inps):
     return assert_fn
 
 
+@jax_funcify.register(PosDefMatrix)
+def jax_funcify_PosDefMatrix(op, **kwargs):
+    def posdefmatrix_fn(value, *inps):
+        no_pos_def = jnp.any(jnp.isnan(jnp.linalg.cholesky(value)))
+        return (no_pos_def == jnp.array(False, dtype="bool")).astype(jnp.int_)
```

Comment:
Do we have a better way to negate a statement and convert it to `int`? This smells bad ...

Reply:
The PyTensornOp doesn't return a bool? It seems like it should

Reply:
It returns an integer:

```python
import pytensor.tensor as pt
from pymc.distributions.multivariate import PosDefMatrix

matrix_pos_def = PosDefMatrix()

matrix_pos_def(pt.eye(2)).eval()
>> array(1, dtype=int8)
```

Reply:
Let's check if we can change it to bool. Should need tweaking of `make_node` and `perform` of the original Op

Reply:
Tried to do it in https://github.com/pymc-devs/pymc/pull/6853/commits/013ead3c8b00b7331429f7702a90092ea733e1a7

Reply:
(and tests https://github.com/pymc-devs/pymc/pull/6853/commits/75ca38bdc7f1bf4acee903cfec8b79de3bfa5f10)

Reply:
Then the jax one can/shoud return the boolean variable `not any(...)`

Reply:
In https://github.com/pymc-devs/pymc/pull/6853/commits/b9a2ef8007bfcce9c166858addf909b3c3bce878 I made the output boolean (as in the tests)